app/services/allocations/creator.rb

```diff
@@ -2,10 +2,9 @@
 
 module Allocations
   class Creator
-    attr_accessor :doorkeeper_application_owner, :allocation_params, :complex_case_params, :allocation
+    attr_accessor :allocation_params, :complex_case_params, :allocation
 
-    def initialize(doorkeeper_application_owner:, allocation_params:, complex_case_params:)
-      self.doorkeeper_application_owner = doorkeeper_application_owner
+    def initialize(allocation_params:, complex_case_params:)
       self.allocation_params = allocation_params
       self.complex_case_params = complex_case_params
```

Comment:
meanwhile could you change these to:
```suggestion
      @allocation_params = allocation_params
      @complex_case_params = complex_case_params
```
it just sounds right with instance variables

Reply:
I replied to this suggestion before. We have the same established pattern in almost every service so I don't think this should be changed.

Reply:
really? I thought it was a small minority, ok no problem